packages/dev-cli/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@graphql-mesh/dev-cli",
+  "version": "0.0.0",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "Urigo/graphql-mesh",
+    "directory": "packages/dev-cli"
+  },
+  "license": "MIT",
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "bin": {
+    "mesh-dev": "dist/cjs/bin.js"
+  },
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "exports": {
+    ".": {
+      "require": {
+        "types": "./dist/typings/index.d.cts",
+        "default": "./dist/cjs/index.js"
+      },
+      "import": {
+        "types": "./dist/typings/index.d.ts",
+        "default": "./dist/esm/index.js"
+      },
+      "default": {
+        "types": "./dist/typings/index.d.ts",
+        "default": "./dist/esm/index.js"
+      }
+    },
+    "./package.json": "./package.json"
+  },
+  "typings": "dist/typings/index.d.ts",
+  "peerDependencies": {
+    "graphql": "*"
+  },
+  "dependencies": {
+    "@graphql-mesh/fusion-composition": "^0.0.0",
+    "@graphql-tools/load-files": "^7.0.0",
+    "@graphql-tools/utils": "^10.0.8",
+    "dotenv": "^16.3.1",
+    "spinnies": "^0.5.1",
+    "ts-node": "^10.9.1"
```

Comment:
this probably needs to be a dev dep? why do we need it at runtime?

Reply:
And if you need it at runtime, for the config file, maybe we should use a more clean dependency than ts-node? it's heavy and not ideal

Reply:
These are dependencies of Development CLI that helps you build a supergraph SDL then you can run it either with Runtime/Serve CLI or Runtime library programmatically.